src/sparse/impl/KokkosSparse_gauss_seidel_spec.hpp

```diff
@@ -110,19 +110,19 @@ namespace KokkosSparse {
                                             KokkosKernels::Experimental::KokkosKernelsHandle< \
                                                                                              const OFFSET_TYPE, const ORDINAL_TYPE, const SCALAR_TYPE, \
                                                                                              EXEC_SPACE_TYPE, MEM_SPACE_TYPE, SLOW_MEM_SPACE> , \
-                                            Kokkos::View<const OFFSET_TYPE *, LAYOUT_TYPE, \
+                                            Kokkos::View<const OFFSET_TYPE *, Kokkos::LayoutLeft, \
                                                          Kokkos::Device<EXEC_SPACE_TYPE, MEM_SPACE_TYPE>, \
                                                          Kokkos::MemoryTraits<Kokkos::Unmanaged> >, \
-                                            Kokkos::View<const ORDINAL_TYPE *, LAYOUT_TYPE, \
+                                            Kokkos::View<const ORDINAL_TYPE *, Kokkos::LayoutLeft, \
                                                          Kokkos::Device<EXEC_SPACE_TYPE, MEM_SPACE_TYPE>, \
```

Comment:
I know this isn't your problem :) but the only operations that need to know about the memory space are those that allocate Views.  If something doesn't allocate a View, we should just treat all Views as nonowning AnonymousSpace.